snorkel/slicing/slicing_classifier.py

```diff
@@ -0,0 +1,176 @@
+from typing import Any, Dict, List, Optional, Set, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from snorkel.analysis import Scorer
+from snorkel.classification import DictDataLoader, DictDataset, Operation, Task
+from snorkel.classification.multitask_classifier import MultitaskClassifier
+
+from .utils import add_slice_labels, convert_to_slice_tasks
+
+
+class BinarySlicingClassifier(MultitaskClassifier):
+    """A slice-aware binary classifier that supports training + scoring on slice labels.
+
+    Parameters
+    ----------
+    base_architecture
+        A network architecture that accepts input data and outputs a representation
+    head_dim
+        Output feature dimension of the representation_net, and input dimension of the
```

Comment:
repr_net to base_arch. use backticks to indicate it's a specific reference

Reply:
👍